master-firmware/src/scara/lie_groups.c

```diff
@@ -0,0 +1,58 @@
+#include "lie_groups.h"
+
+so2_t so2(float angle)
```

Comment:
Maybe we should rename this to `so2_create `to make it easier to find ?

Reply:
Sure. I miss my C++ constructors :stuck_out_tongue: